api/src/main/resources/openmrs-distro.properties

```diff
@@ -0,0 +1,12 @@
+# SDK server properties file
+# import the UgandaEMR distro settings
+
+name=UgandaEMRPOC
```

Comment:
@slubwama We do not need a distro.properties file since this is not a separate distribution, rather just a module included in UgandaEMR which is not started by default